django/contrib/admin/templates/admin/pagination.html

```diff
@@ -1,12 +1,15 @@
 {% load admin_list %}
 {% load i18n %}
-<p class="paginator">
-{% if pagination_required %}
-{% for i in page_range %}
-    {% paginator_number cl i %}
-{% endfor %}
-{% endif %}
+<nav class="paginator" aria-labelledby="pagination">
+    <h2 id="pagination" class="visually-hidden">{% blocktranslate with name=cl.opts.verbose_name_plural %}Pagination {{ name }}{% endblocktranslate %}</h2>
+    <ul>
+    {% if pagination_required %}
+    {% for i in page_range %}
+        <li>{% paginator_number cl i %}</li>
+    {% endfor %}
+    {% endif %}
+    </ul>
 {{ cl.result_count }} {% if cl.result_count == 1 %}{{ cl.opts.verbose_name }}{% else %}{{ cl.opts.verbose_name_plural }}{% endif %}
 {% if show_all_url %}<a href="{{ show_all_url }}" class="showall">{% translate 'Show all' %}</a>{% endif %}
 {% if cl.formset and cl.result_count %}<input type="submit" name="_save" class="default" value="{% translate 'Save' %}">{% endif %}
```

Comment:
All of these elements are now within a `<nav>` for pagination
The one that makes the least sense is the save button for the bulk edit, the others maybe are ok but perhaps are not strictly pagination
I think the nav element should only contain elements for pagination :thinking: